com_dpattachments/admin/src/Controller/AttachmentController.php

```diff
@@ -40,9 +40,13 @@ public function upload(): void
 		try {
 			$model->upload($data);
 
+			$item = $model->getItem($model->getState($model->getName() . '.id'));
+			if( $item->created_by_alias == "" ){
```

Comment:
Better to check here for ->author_name.

Reply:
Yes, it could seem, but at this point, "author_name" is absolutely certain that is not defined since calling to "$model->getItem" returns an "attach" item, which haven't got any property called "author_name", the only option is that "created_by_alias" field is set but there is no option to have an "author_name" property.